lib/linguist/languages.yml

```diff
@@ -4938,6 +4938,16 @@ TOML:
   codemirror_mode: toml
   codemirror_mime_type: text/x-toml
   language_id: 365
+TSX:
```

Comment:
Travis CI builds are failing because TSX should come after TSQL.

Reply:
Oops. You're right. Silly me.